docs/contributor/benchmarks/otlp-logs-validation.md

```diff
@@ -0,0 +1,407 @@
+# OTel LogPipeline Setup Validation
+
+This file documents the process of validating the whole LogPipeline with OTLP output flow. It defines the setup, that consists of the manually deployed log agent, the already-implemented log gateway, and log generators using flog.
+
+The scope is to performance test the agent, observing the resulting values (such as throughput, resource consumption, reaction to backpressure), and to compare the agent to the previous FluentBit-based setup.
+
+
+## Setup Configuration Steps
+
+- To set up the log agent with Helm, run:
+
+    ``` bash
+    k apply -f telemetry-manager/config/samples/operator_v1alpha1_telemetry.yaml
+
+    // Execute knowledge-hub/scripts/create_cls_log_pipeline.sh with the corresponding environment variables 
+
+    helm repo add open-telemetry https://open-telemetry.github.io/opentelemetry-helm-charts
+
+    helm install -n kyma-system logging open-telemetry/opentelemetry-collector -f telemetry-manager/docs/contributor/pocs/assets/otel-log-agent-values.yaml
+    ```
+
+- To set up the log agent manually, run:
+
+    ``` bash
+    k apply -f telemetry-manager/config/samples/operator_v1alpha1_telemetry.yaml
+
+    // Execute knowledge-hub/scripts/create_cls_log_pipeline.sh with the corresponding environment variables 
+
+    k apply -f ./otlp-logs-validation.yaml
+    ```
+
+
+## Relevant/Configurable Resources
+
+- Log Agent ConfigMap (OTel Config)
+- Log Agent DaemonSet
+
+See [OTLP Logs Validation YAML](./otlp-logs-validation.yaml)
+
+#### Things to take into consideration (at implementation)
+- Dynamically include/exclude of namespaces, based on LogPipeline spec attributes.
+- Exclude FluentBit container in OTel configuration, and OTel container in FluentBit configuration.
+- `receivers/filelog/operators`: The copy body to `attributes.original` must be avoided if `dropLogRawBody` flag is enabled.
+
+### How does checkpointing work
+
+> By enabling the storeCheckpoint preset (Helm) the `file_storage` extension is activated in the receiver
```

Comment:
However, adding a full stop makes sense regardless if it's a Note or a normal paragraph.

```suggestion
> By enabling the storeCheckpoint preset (Helm) the `file_storage` extension is activated in the receiver.
```